docs/InstallGuide.md

```diff
@@ -131,294 +158,206 @@ To automatically launch the mysql server, execute the following command.
 
     # chkconfig mysqld on
 
-Create Database
+Create database
 
     # mysqladmin -uroot create vnet
     # cd /opt/axsh/openvnet/vnet
     # bundle exec rake db:init
```

Comment:
The bundle command failed because it is not in my PATH. It needs an extra line to add it to PATH. Also make sure that the openvnet ruby is in front of the current PATH, so no other installed ruby is used.

    # PATH=/opt/axsh/openvnet/ruby/bin:$PATH